vhdbuilder/packer/components.json

```diff
@@ -456,7 +456,8 @@
       "versions": [
         "1.2.7",
         "1.4.14",
```

Comment:
ar we still runnign tight on space? 1.4.14 ready for removal?

Reply:
@paulgmiller I have no idea. Normally I always add a new Azure CNI but do not remove any old Azure CNI version for Linux since I do not know whether I can do it. If it needs to be removed, I think that it should be done in a separate PR. How do you think?

Reply:
defaultValue: "https://acs-mirror.azureedge.net/azure-cni/v1.4.14/binaries/azure-vnet-cni-linux-amd64-v1.4.14.tgz"

Turns out we never used 1.4.21 so you could remove that but .14 has to stay. I'd encourage you to remove .21 but won't block on it.

Reply:
@paulgmiller Charlie shared "v1.4.14 can not be removed." since it is the default toggle value.